misc/bazel/workspace.bzl

```diff
@@ -55,3 +56,11 @@ def codeql_workspace(repository_name = "codeql"):
             "https://github.com/bazelbuild/rules_python/archive/refs/tags/0.8.1.tar.gz",
         ],
     )
+
+    new_git_repository(
```

Comment:
Using `http_archive` with GitHubs codeload service, i.e. 
https://github.com/facebook/fishhook/archive/aadc161ac3b80db07a9908851839a17ba63a9eb1.zip 
tends to be much faster than using git to pull in the code.

Reply:
You probably need to set the `strip_prefix` option to (presumably) `strip_prefix = "fishhook-aadc161ac3b80db07a9908851839a17ba63a9eb1"` - see the `WORKSPACE.bazel` file in semmle-code for examples where we already do this.

Reply:
It did the trick 🎉